### DISCUSSION TITLE
Search pause possibility, Windows XP

### DISCUSSION THREAD

**Discussion by link0127** [qarmin/czkawka#977] [Q&A] [answered]
Dear Community!

I would like to start using this very promising tool, preferably the GUI version of it but I would have two question;
1) Could you please let me know, whether there is any way to pause the searching when I'm running a picute duplication search?
2) I know it's sound strange but can I run it on Windows XP? 

Background (reason of question);
I'm planning to run an image duplication search on a massive amount of data stored on my NAS. During the night I would like to pause the process because my NAS is really noisy and actually it's close to my bed. I would run the tool on a Windows XP VM. I would use Win XP because my NAS do not have too much HW resource, so an XP is more efficient then other more modern system.

Any comment would be very welcomed,
Thanks in advance!

**Comment by qarmin** [comment, marked as answer]
1. There is no plan to implement pausing/resuming in app because this would complicate app architecture without big benefit.
App was created with cache in mind, so instead of pausing/resuming scan, closing/opening app should work most of the time almost identically(not all steps use cache like e.g. comparing audio fingerprints)

Linux allows to pause/resume app with simple commands
```
kill -STOP ${PID}
kill -CONT ${PID}
```

2. Windows XP support was dropped few years ago and even Rust version required to compile app(1.67.1) not supports it, so there is 0% chance that app will work on this OS.
My main machine runs on Ubuntu and I'm able to test only Windows 10 and latest Mac version on VM(I doubt that app works 100% on Windows 7)

Probably the only way to run app on NAS is to install stripped version of Windows 10 or light Linux OS like Lubuntu  22.04+(GTK 4 is quite new thing)

**Reply by link0127** [reply]
Thank you very much for the very quick and prompt answer, it was really detailed, exactly what I'm looking for, much appreciated! I'm sure that based on that I will find the way to manage this.
Thanks again!
